htlcswitch/interceptable_switch.go

```diff
@@ -486,6 +486,21 @@ func (s *InterceptableSwitch) interceptForward(packet *htlcPacket,
 			return false, nil
 		}
 
+		var customRecordsBlob []byte
+
+		htlc.CustomRecordsBlob.WhenSome(
```

Comment:
Other comment re `ExtraData` also applies here. Will only note it here to avoid duplicate comments.